boa/src/syntax/parser.rs

```diff
@@ -347,6 +350,79 @@ impl Parser {
                     Box::new(block),
                 )))
             }
+            Keyword::Try => {
```

Comment:
We should move this to a new function, and add the ECMAScript reference: https://tc39.es/ecma262/#sec-try-statement

Reply:
I agree, i want to move things into functions in this file

Reply:
This has been done in the last commit.